src/utils/mwApi.js

```diff
@@ -0,0 +1,13 @@
+const defautParams = {
+  format: 'json',
+  formatversion: 2,
+  origin: '*'
+}
+
+export const buildMwApiUrl = (lang, params) => {
+  params = Object.assign({}, defautParams, params)
+  const baseUrl = `https://${lang}.wikipedia.org/w/api.php`
+  return baseUrl + '?' + Object.keys(params).map(p => {
+    return `${p}=${encodeURIComponent(params[p])}`
```

Comment:
sorry I think I merged too fast without checking this properly

because the encoding here, if you search for `wikimedia foundation`, you won't get the result from api.

the api doesn't treat `%20` as a space.